ast/misc.go

```diff
@@ -81,11 +81,9 @@ func (n *ExplainStmt) Accept(v Visitor) (Node, bool) {
 type PrepareStmt struct {
 	stmtNode
 
-	InPrepare bool // true for prepare mode, false for use mode
```

Comment:
There is some usages of "InPrepare". This pr remove it but I can not find the deletions of its usages.